src/test/stateful_test_utils.hpp

```diff
@@ -73,18 +73,18 @@ class DummyStatefulModel {
     const std::string MODEL_PATH = std::filesystem::current_path().u8string() + "/src/test/summator/1/summator.xml";
 
     std::shared_ptr<ov::Model> network;
-    std::shared_ptr<ov::runtime::CompiledModel> execNetwork;
+    std::shared_ptr<ov::runtime::CompiledModel> compiledModel;
 
     const std::string stateName = "state";
 
 public:
     DummyStatefulModel() {
         network = ieCore.read_model(MODEL_PATH);
```

Comment:
Here and in multiple other places `ieCore::read_model` output is saved in a variable called `network`.
Is that okay or should these variables be renamed to `model`?